### PR TITLE
nikd01-sp1

### PR DESCRIPTION
**Pull request by dmitrynikolaev08**
Link to static web: https://esotemp.vse.cz/~nikd01/sp1/

### CODE REVIEW

File: www/sp1/nikd01/pages/contact/index.html [hunk 1]
```diff
@@ -47,18 +48,18 @@ <h1>DMITRII NIKOLAEV</h1>
         </div>
         <h2>My tech stack</h2>
         <div class="tech-list">
-            <a href="../front-end/">> HTML + SCSS</a>
-            <a href="../front-end/">> REACT + TYPESCRIPT</a>
-            <a href="../back-end/">> NODE.JS</a>
-            <a href="../front-end/">> FIGMA</a>
+            <a href="../front-end/">-> HTML + SCSS</a>
```
Comment (nvbach91) [inline www/sp1/nikd01/pages/contact/index.html]: takhle se HTML entita `>` nepise

File: www/sp1/nikd01/index.html [hunk 2]
```diff
@@ -9,18 +9,18 @@
...
-- zvyrazneni aktivni polozky menu
-- externi odkaz _blank
-- odkaz na github bez uzivatele
-- chybi cela jedna stranka (minimalni pocet stranek je 5)
-- chybi verze pro tisk
+- zvyrazneni aktivni polozky menu (opraveno)
+- externi odkaz _blank (opraveno)
+- odkaz na github bez uzivatele (opraveno)
+- chybi cela jedna stranka (minimalni pocet stranek je 5) (opraveno - pridana stranka full stack)
+- chybi verze pro tisk (opraveno, ale bohuzel nejde ve verzi pro tisk zobrazovat gradient)
```
Comment (nvbach91) [inline www/sp1/nikd01/index.html:23]: jak myslite nejde zobrazovat gradient? 
image

na verzi pro tisk hlavne vyreste plytvani papirem

**Review by nvbach91 — CHANGES_REQUESTED** [review]

File: www/sp1/nikd01/pages/contact/index.html (hunk 1 above)
Comment (dmitrynikolaev08) [reply www/sp1/nikd01/pages/contact/index.html]: changed with \&gt; entity

File: www/sp1/nikd01/index.html (hunk 2 above)
Comment (dmitrynikolaev08) [reply www/sp1/nikd01/index.html:23]: I forgot to check the 'background graphics' checkbox, that's why these gradients weren't displayed.
Margins and font-sizes for the print version were also updated in the new commit.

**Review by nvbach91 — APPROVED** [review]

**Pull request merged**